inst/extdata/db/2.0.0/sampledb_database_1.4.3_2.0.0.sql

```diff
@@ -0,0 +1,133 @@
+ALTER TABLE "sample_type" ADD COLUMN "parent_id" INTEGER DEFAULT NULL REFERENCES "sample_type"("id");
+
+--- Add shipped status ---
+
+INSERT OR ROLLBACK INTO "status" (name)
+VALUES
+	("Shipped");
+
+INSERT OR ROLLBACK INTO "state_status_relationship" ("status_id", "state_id", "default") 
+VALUES 
+	(5, 2, FALSE);
+
+
+--- Control Collection ---
+CREATE TABLE IF NOT EXISTS "control_collection" (
+	"id"			INTEGER NOT NULL,
+	"url"			VARCHAR NOT NULL,
+	"metadata"		TEXT,
+
+	PRIMARY KEY("id")
+);
+
+
+ALTER TABLE "study" ADD COLUMN "control_collection_id" REFERENCES "control_collection"("id");
```

Comment:
this should be the other way around, control_collection knows about study, study should not have to know about control_collection